CHANGELOG.md

```diff
@@ -8,14 +8,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 The intended audience of this file is for py42 consumers -- as such, changes that don't affect
 how a consumer would use the library (e.g. adding unit tests, updating documentation, etc) are not captured here.
 
-## Unreleased
-
 ### Added
 
 - `code42 audit-logs` commands:
     - `search` to search for audit-logs.
     - `send-to` to send audit-logs to server.
 
+## Unreleased
```

Comment:
unreleased hoist

Reply:
I think this got a bit weird because at some point I merged in additional changes from the main branch. How should I organize things here? Should the new changes be in "added" or "unreleased"?

Reply:
At the moment, you would just need to delete the unreleased and put back the one that was deleted at the top.

However, we plan on cutting a release of audit logs before merging anything else, soon, so you could probably skip the CHANGELOG until the end